articles/api-auth/tutorials/verify-access-token.md

```diff
@@ -0,0 +1,135 @@
+- For `RS256`, the tenant's JSON Web Key Set (JWKS) is used. Your tenant's JWKS is `https://${account.namespace}/.well-known/jwks.json`.
+
+The most secure practice, and our recommendation, is to use `RS256`.
+
+### How can I verify the signature?
+
+To verify a token's signature, you can use one of the libraries listed in the _Libraries for Token Signing/Verification_ section of [JWT.io](https://jwt.io/).
+
+Following the Node.js example of the previous section, the [jwt.verify()](https://github.com/auth0/node-jsonwebtoken#jwtverifytoken-secretorpublickey-options-callback) method of the [node-jsonwebtoken library](https://github.com/auth0/node-jsonwebtoken), supports an `secretOrPublicKey` argument, that should be populated with a string or buffer containing either the secret (for `HS256`), or the PEM encoded public key (for `RS256`). If the verification fails you will get a `invalid signature` error.
+
+## Validate the Claims
+
+Once the API verifies the token's signature, the next step is to validate the standard claims of the token's payload. The following validations need to be made:
+
+- _Token expiration_: The current date/time _must_ be before the expiration date/time listed in the `exp` claim (which is a Unix timestamp). If not, the request must be rejected.
+
+- _Token issuer_: The `iss` claim denotes the issuer of the JWT. The value _must_ match the one configured in your API. For JWTs issued by Auth0, `iss` holds your Auth0 domain with a `https://` prefix and a `/` suffix: `https://${account.namespace}/`.
+
+- _Token audience_: The `aud` claim identifies the recipients that the JWT is intended for. For JWTs issued by Auth0, `aud` holds the unique identifier of the target API (field __Identifier__ at your [API's Settings](${manage_url}/#/apis)). If the API is not the intended audience of the JWT, it _must_ reject the request.
+
+### How can I validate the claims?
+
+To validate the claims, you have to decode the JWT, retrieve the claims (`exp`, `iss`, `aud`) and validate their values.
+
+The easiest way however, is to use one of the libraries listed in the _Libraries for Token Signing/Verification_ section of [JWT.io](https://jwt.io/). Note that not all libraries validate all the claims. In [JWT.io](https://jwt.io/) you can see which validations each library supports (look for the green check marks).
+
+Following the Node.js example, the [jwt.verify()](https://github.com/auth0/node-jsonwebtoken#jwtverifytoken-secretorpublickey-options-callback) method of the [node-jsonwebtoken library](https://github.com/auth0/node-jsonwebtoken), validates these claims, depending on the input arguments:
+- `audience`: if you want to check `aud`, provide a value here
+- `issuer`: string or array of strings of valid values for the `iss` field
+- `ignoreExpiration`: if `true` do not validate the expiration of the token
```

Comment:
We are advising for best practice, there are not options here.

`audience` has to be set to the API identifier
`ignoreExpiration` *must* be set to `false`.